docs/markdown/GettingStarted_UI.md

```diff
@@ -0,0 +1,89 @@
+# The Inviwo Interface
```

Comment:
Great!

Reply:
oh that was actually not even supposed to be in there :D
I'll make a separate PR for the markdown files